src/config/pragmas/tables.js

```diff
@@ -1,42 +1,59 @@
-module.exports = function configureTables ({ arc }) {
+let is = require('../../lib/is')
+let validate = require('./validate')
+
+module.exports = function configureTables ({ arc, errors }) {
   if (!arc.tables || !arc.tables.length) return null
 
-  let tables = arc.tables.map(t => {
-    if (typeof t === 'object' && !Array.isArray(t)) {
-      let name = Object.keys(t)[0]
-      let table = {
-        name,
-        partitionKey: null,
-        partitionKeyType: null,
-        sortKey: null,
-        sortKeyType: null,
-        stream: null,
-        ttl: null,
-        encrypt: null,
-        PointInTimeRecovery: null,
-      }
-      Object.entries(t[name]).forEach(([ key, value ]) => {
+  let isPrimaryKey = val => val.startsWith('*String') || val.startsWith('*Number')
+  let isSortKey = val => val.startsWith('**String') || val.startsWith('**Number')
+
+  let tables = arc.tables.map(table => {
+    if (is.object(table)) {
+      let name = Object.keys(table)[0]
+      let partitionKey = null
+      let partitionKeyType = null
+      let sortKey = null
+      let sortKeyType = null
+      let stream = null
+      let ttl = null
+      let encrypt = null
+      let PointInTimeRecovery = null
+      let legacy
+      let pitr = 'PointInTimeRecovery' // It's just so long
```

Comment:
Can we lift this to the global scope so we don't redefine on every invocation?

Reply:
Four lines highlighted here, I assume you just mean `pitr`? Moved up a level of scope.

Reply:
Correct, `pitr`

Reply:
Got it, and resolved #1155 while I was in there!